pom.xml

```diff
@@ -119,7 +119,7 @@
 			<plugin>
 				<groupId>com.cosium.code</groupId>
 				<artifactId>git-code-format-maven-plugin</artifactId>
```

Comment:
Shouldn't https://github.com/Cosium/git-code-format-maven-plugin#breaking-changes-between-3x-and-4x be addressed?

Reply:
Good catch. Wondering why it didn't break locally. Perhaps default config and ignored props paired with insufficent logs. Will fix that and give you a ping once done.